test/nullable.jl

```diff
@@ -1,5 +1,7 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
+import Base.Dates: Millisecond
```

Comment:
I think it reads best if it is all on one line. I'll remove the import and just go over the 92 character guideline.